packages/radix-ui-themes/.eslintrc.cjs

```diff
@@ -2,4 +2,22 @@ module.exports = {
   root: true,
   extends: ['custom', 'plugin:require-extensions/recommended'],
   plugins: ['require-extensions'],
+  rules: {
+    'import/order': [
+      'error',
+      {
+        'newlines-between': 'always',
+        groups: [['builtin', 'external'], ['parent', 'sibling', 'index'], 'type'],
+      },
+    ],
```

Comment:
This groups imports and provides a stable order, adds a newline before type imports